jwql/tests/test_bad_pixel_monitor.py

```diff
@@ -163,6 +163,7 @@ def test_get_possible_apertures(instrument, expected_list):
     assert ap_list == expected_list
 
 
+@pytest.mark.skipif(ON_GITHUB_ACTIONS, reason='Requires access to central storage.')
```

Comment:
Curious at what happened here, does using the django models not allow us to identify what tables we need to use? How did we do it before?

Reply:
So basically the issue is that, if we want to import the django models (anything under jwql.website.apps.jwql.monitor_models), django will try to run setup in order to configure itself, and it can't do that without a config.json file. So we haven't changed the way that we select what objects to use, it's just that the sqlalchemy model doesn't need django to be set up in order to import it, but the django model does, so the identify tables function, which chooses which table objects to use based on what instrument is being looked at, can't run because the tables themselves haven't been imported.

Reply:
Ah, using the config file 😎